go-gorilla/src/internal/db/db_methods.go

```diff
@@ -159,30 +158,19 @@ func GetFollowing(userID string, limit int) ([]map[interface{}]interface{}, erro
 		return nil, config.DB.Error
 	}
 
-	// Convert []models.Users to []map[interface{}]interface{}
-	var result []map[interface{}]interface{}
-	for _, user := range users {
-		m := map[interface{}]interface{}{
-			"UserID": user.UserID,
-			"Name":   user.Username,
-			"Email":  user.Email,
-			// Add more fields as needed
-		}
-		result = append(result, m)
-	}
-
-	return result, nil
+	return users, nil
 }
 
 // adds a new message to the database
 func AddMessage(text string, author_id int) error {
-	currentTime := time.Now().UTC()
-
-	fmt.Println("timestamp:", currentTime)
+	currentTime, err := time.Parse(time.RFC3339, time.Now().UTC().Format(time.RFC3339))
+	if err != nil {
+		return fmt.Errorf("error formatting time: %v", err)
+	}
 
 	newMessage := models.Messages{
 		AuthorID: author_id,
-		Text:     text,
+		Content:  text,
```

Comment:
Why this? Content x Type?

Reply:
changed back to Text

Reply:
Content is needed here in order for the test_api_endpoints to pass.